layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
```

Comment:
This can be C-l.
^l is form feed and is the default binding for clearing current screen.

the current binding is understandable enough though.

Reply:
After thinking about this again, we probably should use more cubersome key binding than "c".

a single key is easy to press, and is easy to press by mistake.
Clearing the repl seems like a "destructive" action, so we'd better make the key binding idiot-proof.

"C-l" would be the standard binding for clearing screen, but you may use "C" ("s-c").